.gitignore

```diff
@@ -55,7 +55,9 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
+.env*
```

Comment:
I noticed that the `.env` files that are created were not being ignored in commits because they didn't match the `.env` pattern, so I updated this and then added negating patterns for the sample files below.